### PR TITLE
Update SwiftUI application to match UI of other templates

### PR DESCRIPTION
**Pull request by donato-fiore**
What does this implement/fix? Explain your changes.
---------------------------------------------------
This PR adds a application_swiftui template.

Does this close any currently open issues?
------------------------------------------
No

Any relevant logs, error output, etc?
-------------------------------------

No

Any other comments?
-------------------
No

Where has this been tested?
---------------------------
**Operating System:** WSL - Ubuntu 20.04.5

**Platform:** Linux

**Target Platform:** iOS

**Toolchain Version:** Latest

**SDK Version:** iOS 14.5

### CODE REVIEW

**Comment by Luki120** [comment]
[Bruh](https://github.com/theos/templates/pull/24)

**Comment by iCrazeiOS** [comment]
:trollface:

**Comment by evelyneee** [comment]
lgtm 🤝

**Comment by leptos-null** [comment]
As Luki120 noted, this is somewhat a duplicate of #24 (which is now merged).
I like that this PR has a UI that matches the other two `ios/application` templates.
This PR also exposes the small discrepancy that

https://github.com/theos/templates/blob/27e775431377693b9f9e4494cf889d643345461a/ios/application_swiftui/Makefile#L4

should be `include @@THEOS@@/makefiles/common.mk` in this repository.

I think we can either re-purpose this PR to adjust the `Makefile` and `ContentView`, or close this PR.
@donato-fiore What do you think?

File: ios/application_swiftui/ContentView.swift [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
+            }
+        }
+    }
+
+    func delete(at offsets: IndexSet) {
+        items.remove(atOffsets: offsets)
+    }
+}
+
+struct ContentView_Previews: PreviewProvider {
```
Comment (leptos-null) [inline ios/application_swiftui/ContentView.swift:41]: Do we want previews in a Theos template?

File: ios/application_swiftui/ContentView.swift [hunk 2]
```diff
@@ -1,8 +1,45 @@
 import SwiftUI
 
 struct ContentView: View {
-	var body: some View {
-		Text("Hello, world!")
-			.padding()
-	}
+    @State private var items = [Date]()
```
Comment (leptos-null) [inline ios/application_swiftui/ContentView.swift:4]: nit: The `iphone/application_swift` template uses the following syntax:

https://github.com/theos/templates/blob/78f54b96aa0018bb7bd30000965cc07b403ab576/ios/application_swift/RootViewController.swift#L5

Do we have a preference?

File: ios/application_swiftui/ContentView.swift [hunk 3]
```diff
@@ -1,8 +1,45 @@
...
+                .listStyle(InsetGroupedListStyle())
+            }
+            .navigationTitle("Content View")
+            .toolbar {
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    Button(action: {
+                        withAnimation(.easeInOut) {
+                            items.insert(Date(), at: 0)
+                        }
+                    }) {
```
Comment (leptos-null) [inline ios/application_swiftui/ContentView.swift:28]: nit: Use trailing closure syntax

```suggestion
                    Button {
                        withAnimation(.easeInOut) {
                            items.insert(Date(), at: 0)
                        }
                    } label: {
```

**Comment by L1ghtmann** [comment]
Did we want to adjust this further or is it good to go?

File: ios/application_swiftui/ContentView.swift (hunk 2 above)
Comment (leptos-null) [reply ios/application_swiftui/ContentView.swift:4]: Let's use matching syntax for consistency

```suggestion
    @State private var items: [Date] = []
```

File: ios/application_swiftui/ContentView.swift (hunk 1 above)
Comment (leptos-null) [reply ios/application_swiftui/ContentView.swift]: Luki suggested we remove this.
I agree.

**Review by leptos-null — APPROVED** [review]
Thanks for the bump Lightmann.

I've closed all open comments, and tested building the latest template on macOS with `Apple clang version 16.0.0 (clang-1600.0.26.3)` for `arm64` and `arm64e`.

**Pull request merged**